package.json

```
@@ -1,18 +1,48 @@
 {
-	"private": true,
-	"name": "o-colors",
+	"name": "@financial-times/o-colors",
+	"type": "module",
+	"version": "0.0.0",
+	"description": "The default colour palette for all FT products. The palette supports colour contrast checking, colour mixing and toneing.",
+	"keywords": [
+		"o-colours",
+		"colours",
+		"themes",
+		"schemes",
+		"pink",
+		"salmon",
+		"slate",
+		"tone",
+		"mix",
+		"background",
+		"palette"
+	],
+	"homepage": "https://registry.origami.ft.com/components/o-colors",
+	"bugs": {
+		"url": "https://registry.origami.ft.com/components/o-colors"
+	},
+	"license": "MIT",
+	"engines": {
+		"npm": "^7"
+	},
 	"devDependencies": {
 		"eslint": "^7.8.1",
 		"eslint-config-origami-component": "^2.1.0",
-		"origami-ci-tools": "^2.0.2",
 		"remark": "^13.0.0",
 		"remark-lint": "^8.0.0",
-		"remark-preset-lint-origami-component": "^1.2.0",
+		"remark-preset-lint-origami-component": "prerelease",
```

```suggestion
		"remark-preset-lint-origami-component": "^2.0.0",
```

and package-lock.json will be updated